### DISCUSSION TITLE
[Gen 6-ORAS] "Times used area/play/dex/buzz-nav" doesn't increment by 1 when I use go on them. Maybe the numbers aren't the records?

### DISCUSSION THREAD

**Discussion by sp11111111** [kwsch/PKHeX#4339] [Q&A]
**Describe the bug**
"Times used area/play/dex/buzz-nav" doesn't increment by 1 when I use go on them. Maybe the numbers that represent them aren't the records? I have 16 for "Times areanav used", 49 for "Times dexnav used", 1 for "Times buzznav used", and 12 for "Times playnav used." Maybe these numbers aren't those records? Could you look into this?

**To Reproduce**
1. Go on area/buzz/play/dex-nav in-game in an Omega Ruby save file
2. Save the game
3. Check the "Times used area/play/dex/buzz-nav" trainer records in pkhex
4. Compare to the last saved save file. the "Times used area/play/dex/buzz-nav" trainer records didn't each increment by 1. 

**Expected behavior**
"Times used area/play/dex/buzz-nav" trainer records should go up each by 1 or however many times you used area/buzz/play/dex-nav.

**Screenshots**
![2024-08-11-17-36-30-msedge-Exvm0UpCrR](https://github.com/user-attachments/assets/57e471a7-0eb4-460b-8540-9b211502cc41)

**Additional context**
There are no additional context

**Comment by kwsch** [comment]
They are for them, they do not increment when you flip to them, only when they're activating.

**Comment by sp11111111** [comment]
i wanted to see what increments "times used dexnav" by 1 so I went in the game and flipped to dexnav then i walked around for a bit until a pokemon popped up in the tall grass. then i sneaked steps to the pokemon then the search was complete on the dexnav as i encountered the said pokemon. the pokemon was electrike in route 110. after that, i captured the pokemon even though fleeing from it wouldn't have made any difference. after the battle, i saved and then opened the save file in pkhex. i then scrolled down to the trainer record "times used dexnav" and compared to the last save file but to my surprise, the numbers for this record for both save files were the same. i was wondering what increments "times used dexnav" by 1. what did you mean by activation of these -nav's?

**Reply by WonderSquid** [reply]
Meaning you need to open them up personally and choose to use them. Passive encounters don't count towards the tally. Additionally, multiple uses while you still have it open the one time do not count as multiple instances used. The other three are similar and require their features to be actively used by the player rather than flipped to and delivering the data on that initial screen. These are the correct records.
